src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -10,35 +10,36 @@ import "./RoomAttendance.scss";
 type RoomAttendanceProps = {
   venue: PartyMapVenue;
   room: Room;
+  maxVisible?: number;
 };
 
 const MAX_AVATARS_VISIBLE = 2;
```

Comment:
[NB] This can be moved to `settings.ts`

Reply:
Good choice, I agree.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1208/commits/39516c38d1632bac156701a29bcd7644b789de24